src/containerapp/azext_containerapp/_sdk_models.py

```diff
@@ -9263,6 +9263,494 @@ def __init__(
+            outbound_policy: Optional["_models.OutboundPolicy"] = None,
+            **kwargs: Any
+    ) -> None:
+        """
+        :keyword inbound_policy: Optional. Inbound Policy.
+        :paramtype ~azure.mgmt.appcontainers.models.InboundPolicy
+        :keyword outbound_policy: Optional. Outbound Policy.
+        :paramtype ~azure.mgmt.appcontainers.models.OutboundPolicy
+        """
+        super().__init__(**kwargs)
+        self.inbound_policy = inbound_policy
+        self.outbound_policy = outbound_policy
+
+
+class InboundPolicy(_serialization.Model):
+    """The inbound policy.
+    :ivar http_retry_policy: HttpRetryPolicy.
+    :vartype http_retry_policy: ~azure.mgmt.appcontainers.models.HttpRetryPolicy
+    :ivar timeout_policy: Timeout Policy.
+    :vartype timeout_policy: ~azure.mgmt.appcontainers.models.TimeoutPolicy
+    """
+
+    _attribute_map = {
+        "timeout_policy": {"key": "timeoutPolicy", "type": "TimeoutPolicy"},
+        "http_retry_policy": {"key": "httpRetryPolicy", "type": "HttpRetryPolicy"},
+    }
+
+    def __init__(
+            self,
+            *,
+            timeout_policy: Optional["_models.TimeoutPolicy"] = None,
+            http_retry_policy: Optional["_models.HttpRetryPolicy"] = None,
+            **kwargs: Any
+    ) -> None:
+        """
+        :keyword timeout_policy: Optional. Timeout Policy.
+        :paramtype ~azure.mgmt.appcontainers.models.TimeoutPolicy
+        :keyword http_retry_policy: Optional. HttpRetryPolicy.
+        :paramtype ~azure.mgmt.appcontainers.models.HttpRetryPolicy
+        """
+        super().__init__(**kwargs) 
+        self.timeout_policy = timeout_policy
+        self.http_retry_policy = http_retry_policy
+
+
+class OutboundPolicy(_serialization.Model):
+    """The outbound policy.
+    :ivar http_retry_policy: HttpRetryPolicy.
+    :vartype http_retry_policy: ~azure.mgmt.appcontainers.models.HttpRetryPolicy
+    :ivar timeout_policy: Timeout Policy.
+    :vartype timeout_policy: ~azure.mgmt.appcontainers.models.TimeoutPolicy
+    """
+
+    _attribute_map = {
+        "timeout_policy": {"key": "timeoutPolicy", "type": "TimeoutPolicy"},
+        "http_retry_policy": {"key": "httpRetryPolicy", "type": "HttpRetryPolicy"},
+    }
+
+    def __init__(
+            self,
+            *,
+            timeout_policy: Optional["_models.TimeoutPolicy"] = None,
+            http_retry_policy: Optional["_models.HttpRetryPolicy"] = None,
+            **kwargs: Any
+    ) -> None:
+        """
+        :keyword timeout_policy: Optional. Timeout Policy.
+        :paramtype ~azure.mgmt.appcontainers.models.TimeoutPolicy
+        :keyword http_retry_policy: Optional. HttpRetryPolicy.
+        :paramtype ~azure.mgmt.appcontainers.models.HttpRetryPolicy
+        """
+        super().__init__(**kwargs)
+        self.timeout_policy = timeout_policy
+        self.http_retry_policy = http_retry_policy
+
+
+class ContainerAppsResiliency(_serialization.Model):
```

Comment:
Please use `class DaprComponentResiliencyPolicy`

please DON'T add the class manually, as I have upgrade the model to 2023-08-01-preview, the classes should already exist.

Reply:
Fixed.